src/components/TopBar.vue

```diff
@@ -61,10 +60,42 @@ const formatWalletBalance = (): String => {
           <span class="default-text"> Ethereum </span>
           <img alt="Chevron Down" src="@/assets/chevronDown.svg" />
         </div>
-        <div class="top-bar-info">
-          <img alt="Account image" src="@/assets/account.svg" />
-          <span class="default-text text-sm">{{ formatWalletAddress() }}</span>
-          <img alt="Chevron Down" src="@/assets/chevronDown.svg" />
+        <div class="flex flex-col">
+          <div class="top-bar-info" @mouseover="menuToggle = true" @mouseout="menuToggle = false" :style="{
```

Comment:
I think that the mouseover and mouseout is not the best practice for this menuToggle because it can cause unexpected behavior (but we can confirm with design team, if you didn't yet. It's better to keep this action on a onclick function.

Reply:
Designer comments:
Please implement as was commented
![image](https://user-images.githubusercontent.com/74625814/208743325-f1936b56-930d-4e10-b74c-6dbf75a6439b.png)